community/values/src/test/java/org/neo4j/values/storable/ValueComparisonTest.java

```diff
@@ -38,6 +40,16 @@
     private Comparator<Value> comparator = Values.COMPARATOR;
 
     private Object[] objs = new Object[]{
+            // POINTS
+            pointValue( CoordinateReferenceSystem.WGS84, -1.0, -1.0 ),
+            pointValue( CoordinateReferenceSystem.WGS84, 1.0, 1.0 ),
+            pointValue( CoordinateReferenceSystem.WGS84, 1.0, 2.0 ),
+            pointValue( CoordinateReferenceSystem.WGS84, 2.0, 1.0 ),
+            pointValue( CoordinateReferenceSystem.Cartesian, -1.0, -1.0 ),
+            pointValue( CoordinateReferenceSystem.Cartesian, 1.0, 1.0 ),
+            pointValue( CoordinateReferenceSystem.Cartesian, 1.0, 2.0 ),
+            pointValue( CoordinateReferenceSystem.Cartesian, 2.0, 1.0 ),
```

Comment:
Just got curious about this work... I think there should be point-arrays here somewhere too?